WordPress/wordcamp.org
# Tools: Update JS packages

Update NPM packages to latest versions. The biggest changes are in eslint, where there are now new i18n rules; and wp-scripts, which now waits to download puppeteer until the e2e tests are run.

	"@wordpress/api-fetch": "3.11.0" -> "3.13.1"
	"@wordpress/components": "9.2.0" -> "9.4.1"
	"@wordpress/date": "3.8.0" -> "3.9.0"
	"@wordpress/element": "2.11.0" -> "2.13.1"
	"@wordpress/eslint-plugin": "4.0.0" -> "5.0.1"
	"@wordpress/scripts": "7.1.0" -> "8.0.1"
	"@wordpress/url": "2.11.0" -> "2.13.0"
	"css-loader": "3.4.2" -> "3.5.2"
	"react-select": "3.0.8" -> "3.1.0"

This also adds the `yarn` build to the repo, [as `yarn` suggests:](https://yarnpkg.com/getting-started/install#per-project-install)

> To be stable, installs need to be run with the same package manager version across environments, otherwise there's a risk we introduce accidental breaking changes between versions

Install yarn globally, `npm install -g yarn`, then `yarn` should use this local version to run — ensuring everyone uses the same tools to prevent any version issues.

### How to test the changes in this Pull Request:

1. Build, linters, and tests on travis should pass
2. Update & build packages: `yarn && yarn workspaces run build`
3. Spot check WC blocks, virtual embeds, etc

There should be no functional change anywhere.
